### PR TITLE
NEW Add Mass action order validate and set to billed

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
# New [*ADD Mass action order*]
[*Long description*]

and picto on liste

### CODE REVIEW

**Comment by BB2A-Anthony** [comment]
@eldy is not ok for you?

File: htdocs/commande/list.php [hunk 1]
```diff
@@ -700,16 +812,29 @@
...
 	$arrayofmassactions = array(
-		'generate_doc'=>$langs->trans("ReGeneratePDF"),
-		'builddoc'=>$langs->trans("PDFMerge"),
-		'cancelorders'=>$langs->trans("Cancel"),
-		'presend'=>$langs->trans("SendByMail"),
+		'generate_doc'=>img_picto('', 'pdf').'&ensp;'.$langs->trans("ReGeneratePDF"),
+		'builddoc'=>img_picto('', 'pdf').'&ensp;'.$langs->trans("PDFMerge"),
 	);
+	if ($permissiontovalidate) {
+		$arrayofmassactions['prevalidate'] = img_picto('', 'check').'&ensp;'.$langs->trans("Validate");
```
[the remark this answers is not in the document]
Comment (bb2a) [reply htdocs/commande/list.php:819]: Oups sorry i change

[the remark this answers is not in the document]
Comment (bb2a) [reply htdocs/commande/list.php]: @eldy it's change.

**Comment by BB2A-Anthony** [comment]
@eldy it's not ok for you?

**Pull request merged**